requirements.txt

```diff
@@ -2,8 +2,8 @@
 v1pysdk-unofficial==0.4.post4
 
 helga>=1.6.0
-Twisted==13.1.0
-pymongo==2.5.2
-httplib2==0.9.2
-oauth2client==2.0.1
-expiringdict==1.1.3
+Twisted>=13.1.0
+pymongo>=2.5.2
+httplib2>=0.9.2
+oauth2client>=2.0.1
+expiringdict>=1.1.3
```

Comment:
Thanks for checking this out!

py26: Twisted has dropped support for py26, and since helga relies on twisted, this cannot be tested properly. py26 has been EOL > 3 years, and I didn't want to waste the time getting it myself to run the tests so didn't want that liability :). It also has numerous security vulnerabilities, which a basic install of helga will not fix.

unpinned: Just committed to pin all reqs. But my opinion :): there are inevitably conflicts. Helga had a twisted pinned req (since removed) which would conflict with your pinned twisted, just in these two packages we are discussing. Multiple plugins fought with a pinned tweepy in [helga-poems](https://github.com/shaunduncan/helga-poems/commit/99dbbc8eb288c1039ed2943ded24b19c85bd1a66). If pinning per project is needed, something like pyup would also be nice, not sure if you checked that out. In conjunction with travis its pretty cool and would make this less painful. Aside, I'm not a fan of changing and setting up code that MUST be regularly changed because of company specific tooling. Without something like travis+pyup, just for my personal repos I'd spend a crazy amount of time keeping up just with versions. I notice you didn't have helga pinned, probably for the same reason I don't want any of them pinned - they are obsolete eventually, and I don't want to force people eventually to open a PR just for that. At least helga deploys don't touch a lot :). tornado doesn't, requests doesn't, django doesn't. In my cursory glance I couldn't find any popular projects that do :). But a commit is submitted.